app/javascript/Pages/components/Navbar/navbar.scss

```diff
@@ -0,0 +1,174 @@
+.navbar {
+  background: #6600cc;
+  height: 80px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.2rem;
+  position: sticky;
+  top: 0;
+  z-index: 999;
+
+  .navbar-container {
+    display: flex;
+    justify-content: space-between;
+    height: 80px;
+    z-index: 1;
+    width: 100%;
+    max-width: 1300px;
+    margin-right: auto;
+    margin-left: auto;
+    padding-left: 50px;
+
+    .navbar-logo {
+      color: #fff;
+      justify-self: start;
+      cursor: pointer;
+      margin-top: 2%;
+    }
+
+    .navbar-menu-icons {
+      color: #ffffff;
+
+      .navbar-menu-icon-notification {
+        display: flex;
+        cursor: pointer;
+        position: absolute;
+        top: 50%;
+        transform: translate(0, -50%);
+        font-size: 1.8rem;
+        padding: 2rem;
+        right: 0;
+
+        &:hover {
+          font-size: 2rem;
+          transition: all 0.2s ease-out;
+        }
+      }
+      .navbar-menu-icon-burger {
+        display: none;
+      }
+    }
+
+    .nav-menu {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      list-style: none;
+      text-align: center;
+      justify-content: end;
+
+      .nav-item {
+        height: 80px;
+        border-bottom: 2px solid transparent;
+
+        &:hover {
+          border-bottom: 4px solid #ffffff;
+        }
+
+        .nav-links {
+          color: #fff;
+          display: flex;
+          align-items: center;
+          text-decoration: none;
+          padding: 0.5rem 2rem;
+          height: 100%;
+        }
+      }
+
+      .logout {
+        padding: 0.5rem 2rem;
+
+        .btn-logout {
+          border: none;
+          cursor: pointer;
+          white-space: nowrap;
+          background-color: transparent;
+          font-size: 16px;
+          color: #aaaaaa;
+        }
+      }
+    }
+  }
+}
+
+@media screen and (max-width: 960px) {
+  .navbar {
+    .navbar-container {
+      .navbar-logo {
+        position: absolute;
+        top: 0;
+        left: 0;
+        transform: translate(25%, 80%);
+        margin-top: 0;
+      }
+
+      .navbar-menu-icons {
+        display: flex;
+        position: absolute;
+        top: 50%;
+        left: 90%;
+        transform: translate(-100%, -50%);
+        font-size: 1.8rem;
+
+        .navbar-menu-icon-notification {
+          right: 60%;
+        }
+        .navbar-menu-icon-burger {
+          display: flex;
+          cursor: pointer;
+        }
+      }
+
+      .nav-menu {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        height: 92vh;
+        position: absolute;
+        top: 79px;
+        left: -100%;
+        opacity: 1;
+        transition: all 0.5s ease;
+
+        &.active {
+          background: #6600cc;
```

Comment:
**issue (blocking)**: with scss you can have a map (https://sass-lang.com/documentation/values/maps) and get the value of the color you need without repeating the hex code in multiple places